dynamic-demo-plugin/package.json

```diff
@@ -16,9 +16,9 @@
     "@openshift-console/dynamic-plugin-sdk": "file:../frontend/packages/console-dynamic-plugin-sdk/dist/core",
     "@openshift-console/dynamic-plugin-sdk-webpack": "file:../frontend/packages/console-dynamic-plugin-sdk/dist/webpack",
     "@openshift-console/plugin-shared": "file:../frontend/packages/console-plugin-shared/dist",
-    "@patternfly/react-core": "5.1.1",
-    "@patternfly/react-icons": "5.1.1",
-    "@patternfly/react-table": "5.1.1",
+    "@patternfly/react-core": "^6.1.1-prerelease.1",
```

Comment:
Should be changed to same versions as console as part of https://issues.redhat.com/browse/CONSOLE-4464